api/desecapi/models.py

```diff
@@ -219,6 +218,9 @@ def save(self, *args, **kwargs):
 
         self.pdns_sync(new_domain)
 
+    def __str__(self):
```

Comment:
My guess is this is needed for the `StringRelatedField` on the `RRSet` serializer, so we should add a note here that this value must be unique

Reply:
done